tests/example.spec.ts

```diff
@@ -18,6 +18,79 @@ const SAMPLE_OUTPUT_QUERY = `query GetRepository($name: String!, $owner: String!
 
 const SAMPLE_OUTPUT_VARIABLES = `{\n  "name": "name",\n  "owner": "owner"\n}`;
 
+const JSON_TEST_EXPECTED_VARIABLES = `{
+	"orgId": 15,
+	"searchPattern": "232f1350ff01 "
+  }`;
+
+const JSON_TEST_EXPECTED_QUERY = `query GetSuggestions($orgId: ID!, $searchPattern: String!, $contentTypeIds: [ID!]) {
```

Comment:
I'd suggest making this a bit more generic so it doesn't reflect any real-world queries which you might not want to expose... :)

Reply:
Agreed, I'll change as soon as I'm home.